app/src/main/res/xml/settings_preferences.xml

```diff
@@ -23,13 +23,26 @@
         app:iconSpaceReserved="false">
         <ListPreference
             android:entries="@array/download_size_array"
-            android:entryValues="@array/download_size_array"
+            android:entryValues="@array/download_size_array_values"
```

Comment:
Thanks for fixing #198! Would be better to have it in a separate PR, or in this case a commit directly to master without a PR would be also fine I think, as it is a trivial change. "Hiding" it an a PR for another issue is not soo good for maintainability I think.

(not a big deal, so keep it for now but maybe in future do it separately)